plugins/inputs/modbus/modbus_test.go

```diff
@@ -292,15 +383,45 @@ func TestHoldingRegisters(t *testing.T) {
 			write:     []byte{0x88, 0x99, 0xAA, 0xBB, 0xCC, 0xDD, 0xEE, 0xFF},
 			read:      float64(18441921395.520346504),
 		},
+		{
+			name:      "register10_ab_uint16_noScale_noShift",
+			address:   []uint16{10},
+			quantity:  1,
+			byteOrder: "AB",
+			dataType:  "UINT16",
+			write:     []byte{0xAB, 0xCD},
+			read:      uint16(43981),
+		},
+		{
+			name:      "register10_ab_uint16_scale",
+			address:   []uint16{10},
+			quantity:  1,
+			byteOrder: "AB",
+			dataType:  "UINT16",
+			scale:     0.1,
+			write:     []byte{0xAB, 0xCD},
+			read:      uint16(4398),
+		},
+		{
+			name:      "register10_ab_uint16_shift",
+			address:   []uint16{10},
+			quantity:  1,
+			byteOrder: "AB",
+			dataType:  "UINT16",
+			shift:     -50000.0,
+			write:     []byte{0xAB, 0xCD},
+			read:      uint16(0),
+		},
 		{
 			name:      "register10_ab_uint16",
 			address:   []uint16{10},
 			quantity:  1,
 			byteOrder: "AB",
 			dataType:  "UINT16",
 			scale:     1,
+			shift:     12.0,
 			write:     []byte{0xAB, 0xCD},
-			read:      uint16(43981),
+			read:      uint16(43993),
 		},
```

Comment:
Please do not remove existing tests, but rather add new ones. They are there for a reason (in this case to check identity-scaling)!

Reply:
None of the existing tests have been removed, Their names were changed to distinguish them from the added tests. It is a result of git diff that it looks like tests have been altered